src/CMakeLists.txt

```diff
@@ -360,6 +353,13 @@ if (WORLD_INTERFACE STREQUAL "REAL")
   target_link_libraries(LimitSwitchCalibration real_world_interface)
 endif()
 
+add_compile_options(
+  -Wall
+  -Wextra
+  -Wno-deprecated-copy
```

Comment:
Instead of adding this flag we should fix the error that it's reporting. It looks like it caught a legitimate bug, we should have defined the copy assignment operator in `CameraParams`